packages/react/src/__tests__/ReactElementValidator-test.internal.js

```diff
@@ -76,14 +76,16 @@ describe('ReactElementValidator', () => {
 
   it('warns for keys for arrays with no owner or parent info', () => {
     function Anonymous() {
-      return <div />;
+      return React.createElement('div');
```

Comment:
This file was supposed to not use JSX (see the comment at the top). So I fixed this test and moved the JSX-specific tests to the JSX file.